contrib/automation_tests/test_cases/connection_window.py

```diff
@@ -58,6 +59,8 @@ def _execute(self, capture_file_path: str, expect_fail=False):
 
         others_button = self.find_control('Button', '...')
         others_button.click_input()
+        # There is a rare flakyness which causes the file dialog to open slowly: b/242843902
+        time.sleep(10)
```

Comment:
what about adding just 1 second? 10 seconds seems a lot.

Reply:
From the video in the bug it looks as if we would need quite  a bit more time (we don't see the operation finish in ~20s)

Reply:
Yeah, agree, that is way nicer.